src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,6 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_PROPERTY = new Prefix("pr/");
+    public static final Prefix PREFIX_PREFERENCE = new Prefix("pf");
```

Comment:
Suggestion: Better if this follows the "pf/" format

Reply:
Careless mistake. Resolved!